cryostatio/cryostat3
# chore(jmx): remove JMX credentials generation from entrypoint

# Welcome to Cryostat3! 👋
## Before contributing, make sure you have:

**To recreate commits with GPG signature** `git fetch upstream && git rebase --force --gpg-sign upstream/main`
_______________________________________________

Fixes: #461

## Description of the change:
Removes the JMX credentials generation pieces from the container entrypoint script. This was in Cryostat 2.x (probably even 1.0 and before) as a way to allow configuring Cryostat to listen for incoming JMX connections and require authentication. This way, if Cryostat discovered itself by ex. Kubernetes Endpoints query, it would be able to open a network connection to itself.

This is not needed anymore since Cryostat is no longer normally configured to discover itself this way, and if the user wants Cryostat to profile itself then they can create the `localhost:0` custom target.

## How to test:
1. Check out and build PR
2. `./smoktest.bash -O`
3. Cryostat should discover itself with `localhost:0` and `cryostat3:9091`. Go to Event Templates. Both target definitions should work and not require credentials.
